<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeeConnections</class>
 <widget class="QMainWindow" name="SeeConnections">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Connections</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Search By Username</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_username"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Search By Location</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_location"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Search By Profile Language</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_profileLanguage"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Search By Current Company</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_currentCompany"/>
    </item>
    <item>
     <widget class="QPushButton" name="SearchButton">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCommandLinkButton" name="BackButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
